.github/workflows/tail-call.yml

```
@@ -0,0 +1,111 @@
+name: Tail calling interpreter
+on:
+  pull_request:
+    paths:
+      - 'Python/bytecodes.c'
+      - 'Python/generated_tail_call_handlers.c.h'
+  push:
+    paths:
+      - 'Python/bytecodes.c'
+      - 'Python/generated_tail_call_handlers.c.h'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  jit:
+    name: ${{ matrix.target }} (${{ matrix.debug && 'Debug' || 'Release' }})
+    runs-on: ${{ matrix.runner }}
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+# Un-comment as we add support for more platforms for tail-calling interpreters.
+#          - i686-pc-windows-msvc/msvc
+#          - x86_64-pc-windows-msvc/msvc
+#          - aarch64-pc-windows-msvc/msvc
+#          - x86_64-apple-darwin/clang
+#          - aarch64-apple-darwin/clang
+          - x86_64-unknown-linux-gnu/gcc
+#          - aarch64-unknown-linux-gnu/gcc
+        debug:
+          - true
+#          - false
+        llvm:
+          - 19
+        include:
+#          - target: i686-pc-windows-msvc/msvc
+#            architecture: Win32
+#            runner: windows-latest
+#          - target: x86_64-pc-windows-msvc/msvc
+#            architecture: x64
+#            runner: windows-latest
+#          - target: aarch64-pc-windows-msvc/msvc
+#            architecture: ARM64
+#            runner: windows-latest
+#          - target: x86_64-apple-darwin/clang
+#            architecture: x86_64
+#            runner: macos-13
+#          - target: aarch64-apple-darwin/clang
+#            architecture: aarch64
+#            runner: macos-14
```

```suggestion
          - target: x86_64-apple-darwin/clang
            architecture: x86_64
            runner: macos-13
          - target: aarch64-apple-darwin/clang
            architecture: aarch64
            runner: macos-14
```

I guess x86_64 and aarch64 macos is working fine with tail call interpreter at the moment, isn't it?

---

I don't want to test those at the moment. macOS runners are the slowest and also most expensive. There have been discussion about how much CI we are using, and I don't feel like contributing to the macOS queue getting too long.